collector.go

```diff
@@ -26,11 +26,13 @@ type cwCollectorTemplate struct {
 }
 
 type cwCollector struct {
-	Region            string
-	Target            string
-	ScrapeTime        prometheus.Gauge
-	ErroneousRequests prometheus.Counter
-	Template          *cwCollectorTemplate
+	Region							string
```

Comment:
Try to maintain the original indentation to reduce the number of lines that get changed.

Reply:
Done :)